src/ProjectTemplates/Web.ProjectTemplates/content/GrpcService-CSharp/Properties/launchSettings.json

```diff
@@ -3,7 +3,7 @@
     "GrpcService-CSharp": {
       "commandName": "Project",
       "launchBrowser": false,
-      "applicationUrl": "http://localhost:50051",
+      "applicationUrl": "https://localhost:50051",
```

Comment:
Do we want to continue to use a custom port? My vote would be against it

Reply:
So just https://localhost:5001 then?

Reply:
Yes

Reply:
When you talk to Jan next please ask him if there is any reason that gRPC demo apps have ports in 5005x range.